src/dune_cache/local.ml

```diff
@@ -98,15 +67,18 @@ module Artifacts = struct
      If any of the targets couldn't be stored in the temporary directory, then
      the result is [Error] with the corresponding exception. Otherwise, the
      result is [Ok ()]. *)
-  let store_targets_to ~temp_dir ~targets ~mode : unit Or_exn.t =
+  let store_targets_to ~temp_dir ~targets ~(mode : Dune_cache_storage.Mode.t)
+    : unit Or_exn.t
+    =
+    let copy =
```

Comment:
Calling this function `copy` is pretty confusing.
```suggestion
    let hardlink_or_copy =
```

Reply:
Anyway, below is the code I wrote internally. Let's use it here too, to avoid divergence.
```
let store_targets_to ~temp_dir ~targets ~mode : unit Or_exn.t =
  let portable_hard_link_or_copy =
    match (mode : Dune_cache_storage.Mode.t) with
    | Hardlink -> Io.portable_hardlink
    | Copy -> fun ~src ~dst -> Io.copy_file ~src ~dst ()
  in
```

Reply:
Nope, just didn't push the change yet. Pushed it now though.

Reply:
Thanks! Apologies for being too pedantic, but could you use the code snippet I copied above? We've already released that version internally, and it'd be best to keep the diff empty.

Reply:
I copied your code. Btw, `hard_link` is not consistent with `hardlink` which we use everywhere else.

Reply:
Oops, indeed, my bad. Thanks!

Reply:
Feel free to fix this (and I'll fix it internally too).